src/coreclr/jit/gentree.h

```diff
@@ -6905,6 +6905,40 @@ struct GenTreeMDArr : public GenTreeArrCommon
 #endif
 };
 
+struct GenTreeMemmove : public GenTreeOp
```

Comment:
This is a load+store node and so needs some additional handling, e. g. it has to be understood by liveness and interference checking as modifying memory (one can notice it also needs `GTF_ASG` + `GTF_GLOB_REF`). Is there good reason to not make this a new `BlkOpKindUnrolledMemmove` flavor of `STORE_DYN_BLK`?

Reply:
Good idea, addressed